src/ui/components/Rating/index.js

```diff
@@ -66,18 +71,35 @@ export default class Rating extends React.Component {
   }
 
   render() {
-    const { readOnly, size } = this.props;
+    const { i18n, rating, readOnly, size } = this.props;
     const cls = classNames('Rating', {
       'Rating--editable': !readOnly,
       'Rating--small': size === 'small',
```

Comment:
Yeah, that's a fair point. All possible values are defined in the PropType above but that's pretty much useless. I patched it to throw an error. If we were using Flow all of this would be way easier.